secretsanta/cli/cli.py

```diff
@@ -0,0 +1,58 @@
+import click
+import json
+
+from secretsanta.main import funs as secretsanta
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+@click.version_option(version='0.0.1')
+def santa():
+    pass
+
+
+@santa.command()
+@click.argument('json', type=click.Path(exists=True))
+@click.option('--test_run', is_flag=True, help='do a test run with fixed seed')
+def makedict(**kwargs):
+    """
+    Make a randomized dictionary of secret santa assignments.
+
+    JSON is the path to a json file containing the set of participants with names and email addresses.
+    """
+    f_participants = kwargs['json']
+    with open(f_participants, 'r') as f:
+        participants = json.load(f)
+    print(participants)
+    assignments = secretsanta.make_santa_dict(participants, seed=666 if kwargs['test_run'] else None)
+    print(assignments)
```

Comment:
I guess the intention is to log these directly to the command line stdout. Not sure if we want to consider something more fancy with logging here instead (although would postpone to another issue + branch)?

Reply:
not sure if I'd want the CLI example to be logging in a fancier way